src/algorithms/configuration/maps/map.py

```diff
@@ -149,6 +146,7 @@ def get_move_index(self, direction: List[float]) -> int:
         :param direction: The direction
         :return: The index corresponding to :ref:`ALL_POINTS_MOVE_VECTOR`
         """
+        direction = direction.tolist() if torch.is_tensor(direction) else direction
```

Comment:
It would be better to check for `hasattr(direction, "tolist")` since this would then allow for numpy arrays as well.

Reply:
done